### ISSUE TITLE
Add "New version available" log

### ISSUE THREAD

**Issue by szymonrybczak** [callstack/ai-cli#19]
**Is your feature request related to a problem? Please describe.**
There's a bunch of great features in newer versions and it'd be great to inform users that there's a new version available. 

**Describe the solution you'd like**
Add a log when running `ai`, not for every launch because it could be frustrating. 

### Example implementation

```
❯ ai hello

A new release of ai is available: 0.4.0 -> 0.5.0
To upgrade, run: npm update -g @callstack/ai-cli
https://github.com/callstack/ai-cli/releases/tag/v0.5.0

Hello!
```

**Comment by mdjastrzebski** [comment]
@szymonrybczak awesome idea. Would you be able to contribute this as a PR?

**Comment by mdjastrzebski** [comment]
Resolved in #48
